examples/simple.yml

```diff
@@ -0,0 +1,155 @@
+# See https://docs.transcend.io/docs/identity-enrichment
+# Define enricher or pre-flight check webhooks that will be executed
+# prior to privacy request workflows. Some examples may include:
+#   - identity enrichment: look up additional identifiers for that user.
+#                          i.e. map an email address to a user ID
+#   - fraud check: auto-cancel requests if the user is flagged for fraudulent behavior
+#   - customer check: auto-cancel request for some custom business criteria
+enrichers:
+  - title: Basic Identity Enrichment
+    description: Enrich an email address to the userId and phone number
+    input-identifier: email
+    output-identifiers:
+      - userId
+      - phone
+  - title: Fraud Check
+    description: Ensure the email address is not marked as fraudulent
+    input-identifier: email
+    output-identifiers:
+      - email
+  # Note: description is an optional field
+  - title: Analytics Enrichment
+    input-identifier: userId
+    output-identifiers:
+      - advertisingId
+
+# See https://docs.transcend.io/docs/the-data-map#data-silos
+# Define the data silos in your data map. A data silo can be a database,
+# or a web service that may use a collection of different data stores under the hood.
+data-silos:
+  # Note: title is the only required top-level field for a data silo
+  - title: Redshift Data Warehouse
+    description: The mega-warehouse that contains a copy over all SQL backed databases
+    # Specify which data subjects may have personally-identifiable-information (PII) within this system
+    # This field can be omitted, and the default assumption will be that the system may potentially
+    # contain PII for any potential data subject type.
+    data-subjects:
+      - User
+      - Employee
+      - Newsletter Subscriber
+      - B2B Contact
+    # When this data silo implements a privacy request, these are the identifiers
+    # that should be looked up within this system.
+    identity-keys:
+      - email
+      - userId
+    # When a data erasure request is being performed, this data silo should not be deleted from
+    # until all of the following data silos were deleted first. This list can contain other internal
+    # systems defined in this file, as well as any of the SaaS tools connected in your Transcend instance.
+    deletion-dependencies:
+      - Identity Service
```

Comment:
Nitpick: I like the fact that in places like Pulumi, and Serverless we can pass in references to objects directly, without having to specify their name. This doesn't have to be a feature for now, but just a thought that we could do that in the future

Reply:
ya certainly. definitely could see a lot of advantages with more infra as code.